src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6778,6 +6778,15 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ScopedRefAndRefStructOnly" xml:space="preserve">
     <value>The 'scoped' modifier can be used for refs and ref struct values only.</value>
   </data>
+  <data name="ERR_ScopedMismatchInParameterOfOverrideOrImplementation" xml:space="preserve">
+    <value>The 'scoped' declaration of parameter '{0}' doesn't match overridden or implemented member.</value>
```

Comment:
>declaration [](http://example.com/codeflow?start=24&length=11)

It looks like we refer to 'scoped' as modifier in other messages. Can we use the same term here as well?